### PR TITLE
Disable Apply To All button when no other vehicles are selected

### PR DESCRIPTION
**Pull request by mharis001**
**When merged this pull request will:**
- title

### CODE REVIEW

File: addons/garage/stringtable.xml [hunk 1]
```diff
@@ -18,5 +18,8 @@
         <Key ID="STR_ZEN_Garage_ApplyToAll_Tooltip">
             <English>Apply the current vehicle's customization to all selected vehicles of this type.</English>
         </Key>
+        <Key ID="STR_ZEN_Garage_CannotApplyToAll">
+            <English>No other vehicles of the same type selected.</English>
```
Comment (CreepPork) [inline addons/garage/stringtable.xml]: Have been selected.

Comment (neilzar) [reply addons/garage/stringtable.xml]: In the context where the string is used the current form is correct.

Comment (CreepPork) [reply addons/garage/stringtable.xml]: English is not my native language, but to me, it just sounds a tad off.

Comment (neilzar) [reply addons/garage/stringtable.xml]: I'm not saying it as a case of personal preference, I'm saying it as a case of correct grammar. Past perfect is not the correct form to use in this context, there is no sequence of events that would require it. I had a bilingual education, so I should know this.

Comment (jonpas) [reply addons/garage/stringtable.xml]: While I agree, there is something about messages being short and clear, and la language does get in the way sometimes. :)

**Pull request merged**